[
    {
        "number": 1,
        "created_at": "2024-05-19T01:17:00Z",
        "updated_at": "2024-05-19T01:17:00Z",
        "url": "https://api.github.com/repos/fcas/CompreFace/code-scanning/alerts/1",
        "html_url": "https://github.com/fcas/CompreFace/security/code-scanning/1",
        "state": "open",
        "fixed_at": null,
        "dismissed_by": null,
        "dismissed_at": null,
        "dismissed_reason": null,
        "dismissed_comment": null,
        "rule": {
            "id": "py/flask-debug",
            "severity": "error",
            "description": "Flask app is run in debug mode",
            "name": "py/flask-debug",
            "tags": [
                "external/cwe/cwe-215",
                "external/cwe/cwe-489",
                "security"
            ],
            "security_severity_level": "high"
        },
        "tool": {
            "name": "CodeQL",
            "guid": null,
            "version": "2.17.4"
        },
        "most_recent_instance": {
            "ref": "refs/heads/master",
            "analysis_key": ".github/workflows/codeql.yml:analyze",
            "environment": "{\"build-mode\":\"none\",\"language\":\"python\"}",
            "category": "/language:python",
            "state": "open",
            "commit_sha": "def78accd59ad589605ce8017ec8eaa2ef37d309",
            "message": {
                "text": "A Flask app appears to be run in debug mode. This may allow an attacker to run arbitrary code through the debugger."
            },
            "location": {
                "path": "embedding-calculator/src/app.py",
                "start_line": 63,
                "end_line": 63,
                "start_column": 5,
                "end_column": 98
            },
            "classifications": []
        },
        "instances_url": "https://api.github.com/repos/fcas/CompreFace/code-scanning/alerts/1/instances"
    }
]